harness/features/track.local.test.ts

```diff
@@ -0,0 +1,355 @@
+                    const value = 2
+
+                    // const eventType2 = 'textChanged'
+                    const eventType2 = 'textChanged'
+                    const variableId2 = 'json-var'
+                    const value2 = 3
+
+                    const response = await createUser(url, { user_id: validUserId })
+                    await response.json()
+                    const userId = response.headers.get('location')
+
+                    scope
+                        .post(`/client/${clientId}/v1/events/batch`)
+                        .matchHeader('Content-Type', 'application/json')
+                        .reply(519, {})
+
+                    scope.post(`/client/${clientId}/v1/events/batch`).reply((uri, body) => {
+                        eventBody = body
+                        return [201]
+                    })
+
+                    await callTrack(clientId, url, userId,
+                        { type: eventType, target: variableId, value: value })
+                    await callTrack(clientId, url, userId,
+                        { type: eventType2, target: variableId2, value: value2 })
+
+                    await waitForEvent()
+
+                    expect(eventBody).toEqual({
+                        batch: [
+                            {
+                                user: expect.objectContaining({
+                                    platform: 'NodeJS',
+                                    sdkType: 'server',
+                                    user_id: validUserId,
+                                }),
+                                events: [
+                                    expect.objectContaining({
+                                        type: 'customEvent',
+                                        customType: eventType,
+                                        target: variableId,
+                                        value: value,
+                                        user_id: validUserId,
+                                    }),
+                                    expect.objectContaining({
+                                        type: 'customEvent',
+                                        customType: eventType2,
+                                        target: variableId2,
+                                        value: value2,
+                                        user_id: validUserId,
+                                    })
+                                ]
+                            },
+                        ],
+                    })
+                })
+
+                // this test is hard to test for and so the test is changed to ensure that there are 2 flushes
+                // that happen in the roughly specified flush time and also that there are no extra flush that
+                // take place. which is why this test will take longer to run
+                it('should retry with exponential backoff events API call to track 2 events', async () => {
```

Comment:
change the description here since it isn't checking for exponential backoff anymore